src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -6668,6 +6727,41 @@ export class StudyViewPageStore
         );
     }
 
+    @computed
+    get visibleAttributesForClinicalDataTab(): ChartMeta[] {
+        // keep track of clinical attributes in case of duplicates
+        let clinicalAttributes: {
+            [id: string]: ChartMeta;
+        } = {};
+        return _.reduce(
+            Array.from(this._chartVisibility.entries() || []),
+            (acc, [chartUniqueKey, visible]) => {
```

Comment:
put the type in the callback definition instead of down below where you pass the argument.  This will prevent you from using 'as' casting which loosens typing. 
```
_.reduce(acc:ChartMeta[], ...) =>
```